docs/tutorials/start-a-private-network/customchain.md

```diff
@@ -145,93 +155,132 @@ If you enter the command and parameters correctly, the node will return a JSON r
 
 Make sure you delete the file that contains the keys when you are done.
 
+<!-- TODO: add option 3: use the embedded `key` CLI tool -->
+
+### Verify Keys in the Keystore
```

Comment:
@riusricardo this is a new section to review, with a TODO in comments ( a few more predate me working on this) to add more details latter